src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,54 @@
+package lotto.domain;
+
+import java.util.*;
+
+public class Lottos implements Iterable<Lotto>{
+    private List<Lotto> lottoGames = new ArrayList<>();
+
+
```

Comment:
@_@ 빈라인이 있네요!
또한 lottoGames 보단 lottos 더 네이밍에 적당할 것 같은데 어떻게 생각하시나요? 😄 
```suggestion
    private List<Lotto> lottos = new ArrayList<>();

```

Reply:
클래스명은 Lottos라고 변경했는데 미처 인스턴스 변수는 변경하지 못했네요!